sdk/core/System.ClientModel/src/System.ClientModel.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This is showing as including azureicon.png.  Do we want that as part of the beta package?

Reply:
Do you know how to remove that?  It looks like it gets added automatically from here https://github.com/Azure/azure-sdk-for-net/blob/main/eng/Directory.Build.Common.props#L219 which is gated by an `IsClientLibrary` which is false for this project.

Reply:
I do not, but I think it would be worth removing if we want to call this an unbranded package.

Reply:
The engineering system injects it as part of the targets file.  You'd have to remove via a local targets file AFTER doing the load of the next level up so that your action is last in.

Reply:
The initial beta went out with the same

![image](https://github.com/Azure/azure-sdk-for-net/assets/64171366/65cc38a5-af12-485c-b778-80a0730916ef)

I opened this issue to track this https://github.com/Azure/azure-sdk-for-net/issues/40160

Reply:
That beta was for a demo; this was a mistake we made because we were rushing to meet that deadline.

We found the mistake, and we got a new package name, which gives us a change to get it right this time.
Since this is a real release in the official package name and namespace to actual users, I think we should do our best to remove Azure branding from an unbranded library.

Reply:
Luckily i figured out what the issue was I needed to add the remove in .targets not .props so it runs last.

Reply:
See my other comment but I'll add it here as well you should add these properties here in this project https://github.com/Azure/azure-sdk-for-net/blob/main/eng/Directory.Build.Common.props#L154-L155